rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoRuntimeSyntax.scala

```diff
@@ -22,24 +22,25 @@ final class RhoRuntimeOps[F[_]: Sync](
       term: String,
       normalizerEnv: Map[String, Par]
   ): F[EvaluateResult] =
-    evaluate(term, Cost.UNSAFE_MAX, normalizerEnv)
+    evaluate(term, Cost.UNSAFE_MAX, normalizerEnv, Blake2b512Random(128))
 
   def evaluate(
       term: String
   ): F[EvaluateResult] =
-    evaluate(term, Cost.UNSAFE_MAX, Map.empty)
+    evaluate(term, Cost.UNSAFE_MAX, Map.empty, Blake2b512Random(128))
 
   def evaluate(
       term: String,
       initialPhlo: Cost
-  ): F[EvaluateResult] = evaluate(term, initialPhlo, Map.empty)
+  ): F[EvaluateResult] = evaluate(term, initialPhlo, Map.empty, Blake2b512Random(128))
 
   def evaluate(
       term: String,
       initialPhlo: Cost,
-      normalizerEnv: Map[String, Par]
+      normalizerEnv: Map[String, Par],
+      randSeed: Blake2b512Random
   ): F[EvaluateResult] = {
-    implicit val rand: Blake2b512Random = Blake2b512Random(128)
+    implicit val rand: Blake2b512Random = randSeed
```

Comment:
Do we need this change here? We can make it on dev branch to reduce conflict on merging feature branch.

Reply:
https://github.com/rchain/rchain/blob/dev/crypto/src/main/scala/coop/rchain/crypto/hash/Blake2b512Random.scala#L137-L141
This is not a pure function. I can not predict unforgeable name in test without this change.

Reply:
Well. My bad. This change is not neccessary .